src/web/nginx/run_nginx.sh

```diff
@@ -0,0 +1,2 @@
+sed -i "s,/\*INIT_SECTION\*/,window['endpoint']='${API_ENDPOINT}'," /app/index.html
```

Comment:
Why wouldn't we rewrite `constants.ts` instead of embedding it into the html file so it gets built into scripts?

Reply:
Because the constants.ts file is only available at docker image build time before typescript compilation. The file is scrambled after that.

Injecting the api endpoint dependency is deployment and environment specific so it’s better to keep it out of the docker image.

Also, while in ACA world, the endpoint is unlikely to change for a given environment, in a more typical enterprise setting, you would deploy the image to dev then staging then prod or a variation of those and being able to promote the same image without having to rebuild it from env to env is generally a good practice

index.html is a file which stays stable during the build process and can be modified at nginx start time so is a good candidate to perform the injection

What’s more, using a start time environment variable to inject dependencies is a usual approach in Microservices